base/siteorigin-widget.class.php

```diff
@@ -568,7 +569,15 @@ function enqueue_scripts( $form_type = false ) {
 
 			wp_localize_script( 'siteorigin-widget-admin', 'soWidgets', array(
 				'ajaxurl' => wp_nonce_url( admin_url('admin-ajax.php'), 'widgets_action', '_widgets_nonce' ),
-				'sure' => __('Are you sure?', 'so-widgets-bundle')
+				'sure' => __('Are you sure?', 'so-widgets-bundle'),
+				'backup' => array(
+					'newerVersion' => __(
+						'There is a newer version of this widget\'s content available.',
```

Comment:
For translation strings, it's better to avoid slashes inside the string. So in this case, we'd have `"There is a newer version of this widget's content available."`.

Also, I'm not sure if it's important, but I'd always prefer having translation functions all on one line just to make sure it's compatible with anything that parses for translatable strings.